src/io/branch/branchster/SplashActivity.java

```diff
@@ -98,7 +65,41 @@ public void run() {
 	    }.start();
 
 	}
-	
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        branch = Branch.getInstance(mApplicationContext, getResources().getString(R.string.bnc_app_key));
```

Comment:
As an open source demo app, we'd better show user the more formal way to set branch_key (not bnc_app_key) in the strings.xml & manifest.